can_tools/scrapers/official/MT/mt_vaccinations.py

```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 import us
 
-from can_tools.scrapers import CMU
+from can_tools.scrapers import variables as v
```

Comment:
In general we should try to shy away from single letter variables - does `variables` need to be aliased?

Reply:
ah ok, no it doesn't--was just for ease of typing haha, I can remove that

Reply:
thanks!

Reply:
should be fixed!